GoogleChromeLabs/ps-analysis-tool
# Feature: Add tables to PA EE

## Description
This PR adds tables to display interest group and auction config to the explorable explanation tab.

## Relevant Technical Choices



## Testing Instructions
- Clone this branch.
- In the terminal run `npm i && npm run build:ext`.
- Now go to example.com
- Open PSAT from devtools.
- Go to explorable explanation tab.
- As animation progresses, user should be able to see tables switching with the relevant data.

## Additional Information:



## Screenshot/Screencast



---

## Checklist



- ~This code is covered by unit tests to verify that it works as intended.~

Fixes #857
